tuning/data/data_handlers.py

```diff
@@ -137,6 +141,37 @@ def replace_text(match_obj):
     }
 
 
+def apply_custom_data_formatting_jinja_template(
+    element: Dict[str, str],
+    tokenizer: AutoTokenizer,
+    dataset_text_field: str,
+    template: str,
+    **kwargs,
+):
+    """Function to format datasets with jinja templates.
+       Expects to be run as a HF Map API function.
+    Args:
+        element: the HF Dataset element loaded from a JSON or DatasetDict object.
+        dataset_text_field: formatted_dataset_field.
```

Comment:
Please add `tokenizer` to the args here.

Reply:
Also I know this is not on you but can you please fix the doc string for line 104 as well.